src/lib.rs

```diff
@@ -58,3 +58,7 @@ fn my_module(py: Python<'_>, m: &Bound<'_, PyModule>) -> PyResult<()> {
 
     Ok(())
 }
+
+pub(crate) fn convert_to_pybytes(bytes: impl AsRef<[u8]>) -> Py<PyBytes> {
```

Comment:
Do we need the `AsRef` here? Sounds like we might accidentally convert an UTF-8 string to bytes if we don't make this more strict.

Reply:
yes let's make this strict indeed.